cmd/manager/main.go

```diff
@@ -117,10 +114,15 @@ func createManager(leaderElectionNamespace, metricsHost string, metricsPort int3
 		return nil, fmt.Errorf("failed to create a new manager: %w", err)
 	}
 
-	if err := controller.AddDBToTransportSyncers(mgr, postgreSQL, syncService, syncInterval); err != nil {
+	if err := controller.AddDBToTransportSyncers(mgr, postgreSQL, transport, syncInterval); err != nil {
 		return nil, fmt.Errorf("failed to add db syncers: %w", err)
 	}
 
+	err = mgr.Add(transport)
```

Comment:
I think the manager starts the runnables with the same order they were added (cause it uses append if you look internally in the manager code (append always adds to the end).

what will happen if a dbsyncer tries to send data to transport as soon as it starts but transport wasn't started yet?
I don't think we will ever get to this race condition.. but just to ensure correctness, I would add transport to the manager *before* the db syncers.